File: 251004/Marchenko/lab3/Publisher/DTO/ResponseDTO/IssueResponseDTO.cs
using Publisher.Models;

namespace Publisher.DTO.ResponseDTO;

public class IssueResponseDTO
{
    public long Id { get; set; }
    public string Title { get; set; }
    
    public long CreatorId { get; set; }
    public Creator Creator { get; set; }
    
    public string Content { get; set; }
    public DateTime Created { get; set; }
    public DateTime Modified { get; set; }

    public List<Mark> Marks { get; set; } = [];

}

File: 251003/Kapinskiy/Publisher/src/main/java/by/kapinskiy/Publisher/DTOs/Responses/IssueResponseDTO.java
package by.kapinskiy.Publisher.DTOs.Responses;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import java.io.Serializable;
import java.util.Date;
import java.util.List;

@AllArgsConstructor
@NoArgsConstructor
@Getter
@Setter
public class IssueResponseDTO implements Serializable {
    private long id;
    private Long userId;
    private String title;
    private String content;
    private Date created;
    private Date modified;
    private List<String> tags;
}
